pisa/core/binning.py

```diff
@@ -783,7 +783,7 @@ def label(self):
         if self.units == ureg.dimensionless:
             units_tex = ''
         else:
-            units_tex = r' \; \left( {:~L} \right)'.format(self.units)
+            units_tex = r' \; \left[ {:~L} \right]'.format(self.units)
```

Comment:
What is the reason for this change? Most style guides and conventions I'm aware of don't suggest enclosing units in square brackets.